### DISCUSSION TITLE
How to change the default markdown file ( 2022-11-08-10-blog-that-had-gone-way-too-far.md ) name?

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [estruyf/vscode-front-matter#457] [Q&A] [answered]
The front matter headless CMS creates a markdown file name base on `date+title.md`, and my requirement is `title.md`.
example: `2022-11-08-10-blog-that-had-gone-way-too-far.md` and my requirement is `blog-that-had-gone-way-too-far.md`.
Please, anybody, guide me on how to achieve that or where is documentation of that is.

**Comment by estruyf** [comment, marked as answer]
Hi @officialrajdeepsingh, the default behavior is set to have a date prefix. If you want to turn this off, you must set the `frontMatter.templates.prefix` setting with an empty value.

```json
{
	"frontMatter.templates.prefix": ""
}
```

We'll soon let you configure this on the folder level as well. That way, you can, for instance enable it for blog posts, but for other pages turn it off. Discussed here: #452
